clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -13632,82 +13632,36 @@ bool Sema::CheckUsingDeclQualifier(SourceLocation UsingLoc, bool HasTypename,
       RequireCompleteDeclContext(const_cast<CXXScopeSpec&>(SS), NamedContext))
     return true;
 
-  if (getLangOpts().CPlusPlus11) {
-    // C++11 [namespace.udecl]p3:
-    //   In a using-declaration used as a member-declaration, the
-    //   nested-name-specifier shall name a base class of the class
-    //   being defined.
+  // CWG400 [namespace.udecl]p3:
```

Comment:
We should make sure this also matches current wording. I would prefer the commenting to look more like what is in SemaDecl.cpp where we quote the wording and then refer to the core issue at the bottom.